build/tmpl/live-editor.js

```diff
@@ -115,12 +115,12 @@ function program21(depth0,data) {
   stack1 = foundHelper || depth0.imagesDir;
   if(typeof stack1 === functionType) { stack1 = stack1.call(depth0, { hash: {} }); }
   else if(stack1=== undef) { stack1 = helperMissing.call(depth0, "imagesDir", { hash: {} }); }
-  buffer += escapeExpression(stack1) + "/ohnoes-happy.png\"/>\n                </div>\n                <a class=\"error-buddy-thinking\" style=\"display:none;\" href=\"javascript:void()\">\n                    <img src=\"";
+  buffer += escapeExpression(stack1) + "/creatures/ohnoes-happy.png\"/>\n                </div>\n                <a class=\"error-buddy-thinking\" style=\"display:none;\" href=\"javascript:void()\">\n                    <img src=\"";
   foundHelper = helpers.imagesDir;
   stack1 = foundHelper || depth0.imagesDir;
   if(typeof stack1 === functionType) { stack1 = stack1.call(depth0, { hash: {} }); }
   else if(stack1=== undef) { stack1 = helperMissing.call(depth0, "imagesDir", { hash: {} }); }
-  buffer += escapeExpression(stack1) + "/ohnoes-hmm.png\"/>\n                    ";
+  buffer += escapeExpression(stack1) + "/creatures/ohnoes-hmm.png\"/>\n                    ";
```

Comment:
Why is `ohnoes-hmm.png` lowercase here but uppercase in `live-editor.handlebars`?